tasks/shell_crmsh/create-and-push-cib.yml

```
@@ -116,12 +121,16 @@
 # Build the new CIB
 - name: Build the new CIB
   block:
-    ## Cluster properties
-    - name: Configure cluster properties
-      ansible.builtin.include_tasks: crm-cluster-properties.yml
-      vars:
-        properties_set: "{{ ha_cluster_cluster_properties[0] }}"
-      when: ha_cluster_cluster_properties[0].attrs | d([])
+    ## Ensure that stonith is disabled before executing crm configure.
+    ## This is usually disabled by running crm init.
+    ## Executing crm configure without stonith results in "config not valid".
+    - name: Set property stonith-enabled to false
+      ansible.builtin.command:
+        cmd: >-
+          crm -c {{ __ha_cluster_crm_shadow }}
```

```suggestion
          crm -c {{ __ha_cluster_crm_shadow | quote }}
```
not sure if `quote` is really needed here, but if any part of `__ha_cluster_crm_shadow` comes from user input, or can contain shell metacharacters, then the use of `quote` is necessary

---

this is internal variable, already used for shadow CIB commands, so quote is not needed.